src/main/java/org/openrewrite/java/spring/security5/WebSecurityConfigurerAdapter.java

```diff
@@ -144,15 +143,7 @@ public TreeVisitor<?, ExecutionContext> getVisitor() {
                         }
                         // only applicable to former subclasses of WebSecurityConfigurerAdapter - other classes won't be flattened
                         classesToFlatten.add(classDecl);
-                        // Remove imports for annotations being removed together with class declaration
-                        // It is impossible in the general case to tell whether some of these annotations might apply to the bean methods
```

Comment:
Though this comment is true, choosing to not do anything with the annotations, is a wrong move. This can lead to bugs, as I have shown in the description of this PR. It's better/safer to copy them and hope for the best 🤞🏻.